Samples/WindowsCopilotRuntime/cs-maui/README.md

```diff
@@ -0,0 +1,60 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- windows
+- windows-app-sdk
+name: MauiWindowsCopilotRuntimeSample
+description: Shows how to integrate the Windows Copilot Runtime APIs inside MAUI
+urlFragment: cs-maui
+extendedZipContent:
+- path: LICENSE
+  target: LICENSE
+---
+
+## Prerequisites
+- For system requirements, see [System requirements for Windows app development](https://docs.microsoft.com/windows/apps/windows-app-sdk/system-requirements).
+- To ensure your development environment is set up correctly, see [Install tools for developing apps for Windows 10 and Windows 11](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment).
+- Running this sample does require a [Windows Copilot + PC](https://learn.microsoft.com/windows/ai/npu-devices/)
+- This sample requires [Windows App SDK 1.7 Experimental3](https://learn.microsoft.com/windows/apps/windows-app-sdk/downloads#windows-app-sdk-17-experimental), which should be installed by Visual Studio when deploying or launching the app.
```

Comment:
How do we remember to bump this when we release?

Reply:
Currently the WCR samples are only in the `release/experimental` branch. In order to move them to the `main` branch (once WCR goes stable), we'll need to update them to point to the minimum stable build. After that, we theoretically bump all samples in `main` to the latest release after each stable release (not the monthly releases).